### PR TITLE
added class twitter

### PR DESCRIPTION
**Pull request by decobots**
issue  #14

### CODE REVIEW

File: twitter_class.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from environment_variables import get_env
+
+
+class Twitter:
+    method = {"twitter_upload_pic": "https://upload.twitter.com/1.1/media/upload.json",
+              "twitter_create_post": "https://api.twitter.com/1.1/statuses/update.json",
+              "twitter_get_users_posts": "https://api.twitter.com/1.1/statuses/user_timeline.json",
+              "delete_tweet_by_id": "https://api.twitter.com/1.1/statuses/destroy/{}.json"}
+
+    methodType = {"twitter_upload_pic": "post",
```
Comment (livercat) [inline twitter_class.py]: Use snake_case instead of camelCase

File: twitter_class.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                  resource_owner_secret=get_env("TWITTER_ACCESS_SECRET"),signature_type="body")
+
+    def request(self, method_name, **kwargs):
+        params = {}
+        result = None
+        for key in kwargs:
+            params[key] = kwargs[key]
+        if self.methodType[method_name] == "post":
+            if method_name == "delete_tweet_by_id":  # that method has different query structure than others
+                result = requests.post(self.method[method_name].format(kwargs["tweet_id"]), data=params, auth=self.auth)
```
Comment (livercat) [inline twitter_class.py]: You can do data=**kwargs, without intermediary “params”

Comment (livercat) [reply twitter_class.py]: Also, you can do .format on every url - if there are no {} inside, it will have no effect

File: twitter_class.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    def upload_photo(self, name, data):
+        data = base64.b64encode(data)
+        upload_photo_result = self.request("twitter_upload_pic", name=name, media_data=data)
+        return json.loads(upload_photo_result.text)['media_id'], name
+
+    def create_post(self, status, id_of_photo=[]):
+        self.request("twitter_create_post", status=status, media_ids=id_of_photo)
+
+    def get_users_posts(self, number_of_tweets_to_delete):
```
Comment (livercat) [inline twitter_class.py]: number_of_tweets_to_delete looks out of place here

File: twitter_class.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import json
```
Comment (livercat) [inline twitter_class.py]: twitter_class.py -> twitter.py

File: twitter_class.py (hunk 1 above)
Comment (decobots) [reply twitter_class.py]: Done

File: twitter.py [hunk 5]
```diff
@@ -44,8 +39,8 @@ def upload_photo(self, name, data):
     def create_post(self, status, id_of_photo=[]):
         self.request("twitter_create_post", status=status, media_ids=id_of_photo)
 
-    def get_users_posts(self, number_of_tweets_to_delete):
-        get_users_posts_request = self.request("get_users_posts", count=number_of_tweets_to_delete)
+    def get_users_posts(self, number_of_tweets_to_get):
```
Comment (livercat) [inline twitter.py]: Just "amount" will be enough of a name here i think.

**Pull request merged**